src/DashboardService.Data/ChangeLogRepository.cs

```diff
@@ -0,0 +1,45 @@
+﻿using LT.DigitalOffice.DashboardService.Data.Interfaces;
+using LT.DigitalOffice.DashboardService.Data.Provider;
+using LT.DigitalOffice.DashboardService.Models.Db;
+using LT.DigitalOffice.DashboardService.Models.Dto.Requests.ChangeLog.Filter;
+using Microsoft.AspNetCore.JsonPatch;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace LT.DigitalOffice.DashboardService.Data;
+
+public class ChangeLogRepository : IChangeLogRepository
+{
+  private readonly IDataProvider _provider;
+
+  public ChangeLogRepository(IDataProvider provider)
+  {
+    _provider = provider;
+  }
+
+  public Task<Guid?> CreateAsync(DbChangeLog changeLog)
```

Comment:
Add cancellationToken in all async methods

Reply:
++